mrjones2014/yaclt
# Provide a default value for `--validationPattern`

Resolves: #163 

## How to Test

1. Run `yarn build`
2. Run `./dist/index.js validate` -- it should pass
3. In `yacltrc.js` set `validationPattern` to `"\\[.*\\]\\s+.*\\s{#[\\d]+}\s?"` (this is the default but with `\s?` (optional whitespace) appended)
4. Run `./dist/index.js validate` -- it should succeed but log a warning about using the default value for one but not the other
5. In `yacltrc.js` delete the value you added for `validationPattern` and set `format` to `'[{{changeType}}] {{message}} {{echo "{"}}#{{issueId}}{{echo "}"}}  '` (the default with some whitespace added to the end)
6. Run `./dist/index.js validate` -- it should succeed but log a warning about using the default value for one but not the other
